packages/editor-ui/src/constants.ts

```diff
@@ -737,3 +735,6 @@ export const MOUSE_EVENT_BUTTONS = {
 	BROWSER_BACK: 8,
 	BROWSER_FORWARD: 16,
 } as const;
+
+export const WEBSITE_TEMPLATES_URL =
+	'https://n8n.io/workflows/?utm_source=n8n_app&utm_medium=template_library';
```

Comment:
Is the return URL added later on?

Reply:
Yes, we are adding those in phase 3

Reply:
Sorry @tomi, that's not true. We are adding it in this phase. Adding it right now.